cloud_function/pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+env =
+    WFL_URL="https://workflow-launcher.gotc-dev.broadinstitute.org"
```

Comment:
```suggestion
    WFL_URL="https://dev-wfl.gotc-dev.broadinstitute.org"
```

Reply:
workflow-launcher, if it still exists, is using orphaned resources. `dev-wfl` is what is currently managed in gotc-dev.

Reply:
@samanehsan - this is lifted from your documentation. Can you confirm?

Reply:
My mistake! The correct url is `WFL_URL="https://dev-wfl.gotc-dev.broadinstitute.org"`. The tests currently mock everything that use these values, so that's why they didn't fail